src/Multisig.sol

```diff
@@ -233,6 +235,8 @@ contract Multisig is IMultisig, IMembership, PluginUUPSUpgradeable, ProposalUpgr
 
         for (uint256 i; i < _destinationActions.length;) {
             proposal_.destinationActions.push(_destinationActions[i]);
+
+            // Q(danielw): With the new solidity version, using `++i` in the for loop is good enough.
```

Comment:
This change applies for compiler versions equal or greater than 0.8.22
Unfortunately, the dependencies we use force us to stay on version 0.8.17 for now, where such optimization was not present